### DISCUSSION TITLE
Horizontal scrolling

### DISCUSSION THREAD

**Discussion by bellwether-softworks** [emilk/egui#692] [Ideas]
I'm working in layouts that are spanning widths that exceed screen bounds on smaller screens, and don't see any obvious way to handle the clipped regions (not a blocker for me, since users are on wide monitors).  Horizontal scrolling regions would be invaluable for such cases.

**Comment by coderedart** [comment]
this what you looking for?
https://github.com/emilk/egui/issues/391
https://github.com/emilk/egui/pull/663

**Reply by bellwether-softworks** [reply]
I think so! I did try to search for this before commenting, but missed it; thanks!
